### DISCUSSION TITLE
Emphasise take-aways a bit more

### DISCUSSION THREAD

**Discussion by pavlenex** [BitcoinDesign/Guide#371] [Ideas]
Chethan brought this up on Slack. Opening up for a discussion on how we can emphasize the key take-aways a bit more. Any thoughts?

> Every-time we talk about something is really important or is pretty much the takeaway, would be super helpful to emphasise it.
![image](https://user-images.githubusercontent.com/36959754/120688328-3c945700-c4a3-11eb-8dda-845b0a93819e.png)

>  At the end of every page, we can have a few bullet points of the most important things to note. Sort of like a summary or takeaways of what was explained in the page and main points to remember. Helps in quickly getting ready with important things and gives context when moving onto the next page.

**Comment by GBKS** [comment]
This goes back to the idea of check-lists that we have tossed around before. A few thoughts:

1. For how many pages is it possible to extract these short take-aways? Does it work for all types of content or only for ones with specific UI tips?
1. Are there global takeaways for everyone or does every reader have different ones based on their interest?
1. Is this basically our `Do's` and `Don'ts`?

A good next step could be to review a few pages and identify these takeaways. From that, we can more easily decide what an appropriate solution should be.
